### PR TITLE
Add ML-Agents and RL-Baselines3-Zoo

### PR DESCRIPTION
**Pull request by simoninithomas**
Hey there 👋 ,

I've added RL-Baselines3-Zoo and ML-Agents integration to the libraries doc.
I followed the alphabetical order since it's seems to be the classification method.

Edit:
- I also added ML-Agents directly on Librairies.ts and Tasks (thanks for the tip Omar).

### CODE REVIEW

**Comment by osanseviero** [comment]
Thanks for the PR! Would it be possible to also add some high level guides such as https://github.com/huggingface/hub-docs/blob/main/docs/hub/allennlp.md /  https://huggingface.co/docs/hub/allennlp?

**Comment by simoninithomas** [comment]
Sure no pb! I also added ML-Agents in Librairies and const.ts (tasks). Initially I wanted to make a separate PR but I commit on the same branch.

**Comment by osanseviero** [comment]
Up to you! I'm ok with separate PRs :) just let us know when it's ready for review by requesting it

**Comment by simoninithomas** [comment]
The 3 documentations are done 🥳 @osanseviero 
- MLAgents (but one problem with the Gif display 😢 )
- SB3
- RL-Zoo

File: docs/hub/ml-agents.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+`ml-agents` is an open-source toolkit that enables games and simulations made with Unity to serve as environments for training intelligent agents.
+
+## Exploring ML-Agents in the Hub
+
+You can find ml-agents models by filtering at the left of the [models page](https://huggingface.co/models?library=ml-agents).
+
+All models on the Hub come up with useful features:
+1. An automatically generated model card with a description, a training configuration, and more.
+2. Metadata tags that help for discoverability.
+3. Tensorboard summary files to visualize the training metrics.
```
Comment (osanseviero) [inline docs/hub/ml-agents.md:12]: Awesome!

File: docs/hub/ml-agents.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+4. A link to the Spaces web demo where you can visualize your agent playing in your browser.
+
+![ML-Agents demo on Spaces](/docs/assets/ml-agents/ml-agents-demo.gif)
+
+## Install the library
+To install the `ml-agents` library, you need to clone the repo:
+
+```
+# Clone the repository
+git clone https://github.com/huggingface/ml-agents/
```
Comment (osanseviero) [inline docs/hub/ml-agents.md:22]: Is the plan to merge this into ml-agents?

File: docs/hub/ml-agents.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+# Go inside the repository and install the package
+cd ml-agents
+pip3 install -e ./ml-agents-envs
+pip3 install -e ./ml-agents
+```
+
+## Using existing models
+You can simply download a model from the Hub using `mlagents-load-from-hf`
```
Comment (osanseviero) [inline docs/hub/ml-agents.md:31]:
```suggestion
You can simply download a model from the Hub using `mlagents-load-from-hf`.
```

File: docs/hub/rl-baselines3-zoo.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
+# Using RL-Baselines3-Zoo at Hugging Face
+
+`rl-baselines3-zoo` is a training framework for Reinforcement Learning, using Stable Baselines3.
```
Comment (osanseviero) [inline docs/hub/rl-baselines3-zoo.md:3]:
```suggestion
`rl-baselines3-zoo` is a training framework for Reinforcement Learning using Stable Baselines3.
```

File: docs/hub/rl-baselines3-zoo.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+## Exploring RL-Baselines3-Zoo in the Hub
+
+You can find RL-Baselines3-Zoo models by filtering at the left of the [models page](https://huggingface.co/models?library=stable-baselines3).
+
+The Stable-Baselines3 team is hosting a collection of +150 trained Reinforcement Learning agents with tuned hyperparameters that you can find [here](https://huggingface.co/sb3).
+
+All models on the Hub come up with useful features:
+1. An automatically generated model card with a description, a training configuration, and more.
```
Comment (osanseviero) [inline docs/hub/rl-baselines3-zoo.md:12]: Maybe also mention metrics

File: docs/hub/rl-baselines3-zoo.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+python -m utils.load_from_hub --algo ppo --env CartPole-v1 -f logs/ -orga sb3
+```
+
+Else:
+
+```
+# Download model and save it into the logs/ folder
+python -m utils.load_from_hub --algo dqn --env SpaceInvadersNoFrameskip-v4 -orga ThomasSimonini -f logs/
+python enjoy.py --algo dqn --env SpaceInvadersNoFrameskip-v4  -f logs/
+```
```
Comment (osanseviero) [inline docs/hub/rl-baselines3-zoo.md:32]: I think you can merge the two of them as the code is the same, you just change the username/org

File: docs/hub/rl-baselines3-zoo.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+You can define three parameters:
+- `--repo-name`: The name of the repo.
+- `-orga`: Your Hugging Face username.
```
Comment (osanseviero) [inline docs/hub/rl-baselines3-zoo.md:36]:
```suggestion
- `-orga`: A Hugging Face username or organization.
```

File: docs/hub/rl-baselines3-zoo.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+- `-f`: The destination folder.
+
+## Sharing your models
+You can easily upload your models with `push_to_hub`, that will save the model, evaluate it, generate a model card and record a replay video of your agent before pushing the complete repo to the Hub.
```
Comment (osanseviero) [inline docs/hub/rl-baselines3-zoo.md:40]:
```suggestion
You can easily upload your models with `push_to_hub`. That will save the model, evaluate it, generate a model card and record a replay video of your agent before pushing the complete repo to the Hub.
```

File: docs/hub/stable-baselines3.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+
+`stable-baselines3` is a set of reliable implementations of reinforcement learning algorithms in PyTorch.
+
+## Exploring Stable-Baselines3 in the Hub
+
+You can find Stable-Baselines3 models by filtering at the left of the [models page](https://huggingface.co/models?library=stable-baselines3).
+
+All models on the Hub come up with useful features:
+1. An automatically generated model card with a description, a training configuration, and more.
+2. Metadata tags that help for discoverability.
```
Comment (osanseviero) [inline docs/hub/stable-baselines3.md:11]: Maybe also mention metrics

File: js/src/lib/interfaces/Libraries.ts [hunk 10]
```diff
@@ -469,5 +470,11 @@ export const MODEL_LIBRARIES_UI_ELEMENTS: { [key in keyof typeof ModelLibrary]?:
 		repoUrl:  "https://github.com/huggingface/huggingface_sb3",
 		snippet:  stableBaselines3,
 	},
+	"ml-agents": {
+		btnLabel: "ml-agents",
+		repoName: "ml-agents",
+		repoUrl:  "https://github.com/huggingface/ml-agents",
+		snippet:  ml-agents,
```
Comment (osanseviero) [inline js/src/lib/interfaces/Libraries.ts:477]: You are missing the snippet. Check out the others, they are functions with the code snippet to be shown

**Review by osanseviero — COMMENTED** [review]
Looks great! It's only missing the code snippet for ML Agents

File: docs/hub/rl-baselines3-zoo.md (hunk 5 above)
Comment (simoninithomas) [reply docs/hub/rl-baselines3-zoo.md:12]: ✔️

File: docs/hub/rl-baselines3-zoo.md (hunk 6 above)
Comment (simoninithomas) [reply docs/hub/rl-baselines3-zoo.md]: ✔️

File: docs/hub/stable-baselines3.md (hunk 9 above)
Comment (simoninithomas) [reply docs/hub/stable-baselines3.md:11]: ✔️

File: js/src/lib/interfaces/Libraries.ts (hunk 10 above)
Comment (simoninithomas) [reply js/src/lib/interfaces/Libraries.ts]: Added thanks for the tip!

**Comment by simoninithomas** [comment]
I updated based on your feedback @osanseviero though I've still have a problem with the gif when I check the MLAgents page. Is it because it's a temporary build?

**Review by osanseviero — APPROVED** [review]
Nice! Thanks for this :fire:

**Comment by osanseviero** [comment]
@mishig25 do we have support for gifs with docs builder?

**Comment by osanseviero** [comment]
re-ping @mishig25 on this one :)

**Pull request merged**